### PR TITLE
Admin oidc

### PR DESCRIPTION
**Pull request by sbor23**
Use mozilla-django-oidc for django-admin login (button "Login with SSO") along side user/password auth.
This required some settings changes as well as additional settings.
Make sure to check the README!

Some cleanups and deps updates in the process as well.

### CODE REVIEW

File: timed/templates/login.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  </div>
+</form>
+
+{% if user.is_authenticated %}
+  <p>Current user: {{ user.email }}</p>
+  <form action="{% url 'oidc_logout' %}" method="post">
+    <input type="submit" value="logout">
+  </form>
+{% else %}
+  <a href="{% url 'oidc_authentication_init' %}">Login with SSO</a>
```
Comment (winged) [inline timed/templates/login.html:59]: String is not translated

File: timed/templates/login.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <a href="{{ password_reset_url }}">{% trans 'Forgotten your password or username?' %}</a>
+  </div>
+  {% endif %}
+  <div class="submit-row">
+    <label>&nbsp;</label><input type="submit" value="{% trans 'Log in' %}">
+  </div>
+</form>
+
+{% if user.is_authenticated %}
+  <p>Current user: {{ user.email }}</p>
```
Comment (winged) [inline timed/templates/login.html:54]: String is not translated

**Comment by sbor23** [comment]
@winged I put the local login behind a config flag and the strings translated

**Pull request merged**